src/Illuminate/Http/Middleware/PassThroughTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Illuminate\Http\Middleware;
+
+trait PassThroughTrait
+{
+    /**
```

Comment:
Sure @overtrue , maybe I misexplained.
The `$except` property should be moved from `VerifyCsrfToken` to `PassThroughTrait`